modules/react/radio/stories/examples/Basic.tsx

```diff
@@ -12,19 +10,15 @@ export const Basic = () => {
     setValue(value);
   };
 
-  const StyledFormField = styled(FormField)({
-    width: space.xl,
-  });
```

Comment:
The reason it was loosing focus was because this is inside of the render of the component, so with each focus, it would re render and lose focus. If we moved this above line 6, everything should work! I think we want to keep the styled so that we can see the label wrapping or we can just add a style attribute